src/routes/ar-io.ts

```diff
@@ -243,14 +243,14 @@ arIoRouter.post(
         return;
       }
 
-      const parquetExporter = await ParquetExporter.create({
+      const parquetExporter = new ParquetExporter({
```

Comment:
@djwhitt Ideally we would initiate `ParquetExporter` in system so we can stop the worker when stopping the application, but that means we will start a worker even if we never use it.

What do you think about instantiate it here and import this instance to system so we can call `stop` when we stop the application? With that we only start a worker when this endpoint is used.